static_src/components/notification.jsx

```diff
@@ -0,0 +1,80 @@
+
+import React from 'react';
+import keymirror from 'keymirror';
+
+import Action from './action.jsx';
+import createStyler from '../util/create_styler';
+import { entityHealth } from '../constants.js';
+import style from 'cloudgov-style/css/cloudgov-style.css';
+
+const STATUSES = Object.assign({}, entityHealth, keymirror({
+  info: null
+}));
+
+
+const propTypes = {
+  message: React.PropTypes.string,
+  status: React.PropTypes.oneOf(Object.keys(STATUSES)),
+  actions: React.PropTypes.arrayOf(React.PropTypes.object),
```

Comment:
I'm not sure I agree with that as to create each action, it needs an object with a `text` and `clickHandler` fields.